client/src/Components/Townbox/index.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import './index.css'
+import {Button} from '../Button'
+
+export interface TownboxProps {
+  /**
+   * Header colour: enter either orange/primary
+   */
+  headerColor?: 'primary' | 'orange'
+
+  /**
+   * Heading contents
+   */
+  headerText?: string
+
+  /**
+   * Textbox contents
+   */
+  captionText?: string
+
+  /**
+   * Show visited button?
+   */
+  showButton?: boolean
+
+  /**
+   * specify width in terms of px
+   */
+  widthSize?: number
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Townbox: React.FC<TownboxProps> = ({
+  headerText = 'headerText',
+  captionText = 'captionText',
+  headerColor = 'orange',
+  showButton = false,
+  widthSize = 550
+}) => {
+  return (
+    <div className = "flex" style={{width: widthSize}}>
+      
+      <h2 className = {`font-serif text-white p-2 px-4 border-black border-4 rounded-md ml-10 mt-4 absolute z-10 bg-${headerColor}`}
```

Comment:
You shouldn't be using `bg-${headerColor}` since tailwind will not work as intended in production. Define a variable with to deal with this conditional and add it to the list of classes.